helm/prometheus-rules/templates/platform/atlas/recording-rules/monitoring.resource-usage-estimation.rules.yaml

```diff
@@ -1,3 +1,20 @@
+{{- if .Values.mimir.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: monitoring.resource-usage-estimation.recording.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: monitoring.resource-usage-estimation.recording
+    rules:
+    - expr: (count({__name__=~".+"}) by (cluster_id, job) / on(cluster_id) group_left prometheus_tsdb_head_series{instance=~"^.*:9090"}) * on(cluster_id) group_left sum(container_memory_usage_bytes{container="ingester", namespace="mimir"}) by (cluster_id)
```

Comment:
You can add the if only here and not duplicate most of the template :D

Reply:
Does Mimir actually have prometheus_tsdb_head_series ?

Reply:
Yeah you're right !

Reply:
Fixed both.